ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -212,43 +231,43 @@ def collect(self, device, unused):
                 _agent['dhcp_agent_subnets'] = _subnets
                 _agent['dhcp_agent_networks'] = _networks
 
-        result = yield neutron.networks()
-        results['networks'] = result.get('networks', [])
+        _neutron_networks = yield neutron.networks()
+        results['networks'] = _neutron_networks.get('networks', [])
 
-        result = yield neutron.subnets()
-        results['subnets'] = result.get('subnets', [])
+        _neutron_subnets = yield neutron.subnets()
+        results['subnets'] = _neutron_subnets.get('subnets', [])
 
-        result = yield neutron.routers()
-        results['routers'] = result.get('routers', [])
+        _neutron_routers = yield neutron.routers()
+        results['routers'] = _neutron_routers.get('routers', [])
 
-        result = yield neutron.ports()
-        results['ports'] = result.get('ports', [])
+        _neutron_ports = yield neutron.ports()
+        results['ports'] = _neutron_ports.get('ports', [])
 
-        result = yield neutron.floatingips()
-        results['floatingips'] = result.get('floatingips', [])
+        _neutron_floatingips = yield neutron.floatingips()
+        results['floatingips'] = _neutron_floatingips.get('floatingips', [])
 
         # Cinder
         results['cinder_url'] = yield cinder.get_url()
 
-        result = yield cinder.volumes()
-        results['volumes'] = result.get('volumes', [])
+        _cinder_volumes = yield cinder.volumes()
+        results['volumes'] = _cinder_volumes.get('volumes', [])
 
-        result = yield cinder.volumetypes()
-        results['volumetypes'] = result.get('volume_types', [])
+        _cinder_volumetypes = yield cinder.volumetypes()
+        results['volumetypes'] = _cinder_volumetypes.get('volume_types', [])
 
-        result = yield cinder.volumesnapshots()
-        results['volsnapshots'] = result.get('snapshots', [])
+        _cinder_volumesnapshots = yield cinder.volumesnapshots()
```

Comment:
we might want to look into snapshots- we've been bitten by them before (AWS).. it's possible there might be a large number of them and little value (and no monitoring).  We might want to consider disabling them by default.

Reply:
zOpenStackInfrastructureIgnoreSnapshots? (zprops are getting long and longer).

Reply:
Opened a ticket for this in : https://jira.zenoss.com/browse/ZPS-4800 that will add a zOpenStackModelIgnore parameter.